cli/types/cypress.d.ts

```diff
@@ -2633,6 +2633,13 @@ declare namespace Cypress {
     cmdKey: boolean
   }
 
+  interface CookieOptions extends Loggable, Timeoutable {
+    /**
+     * Domain to set cookies on or get cookies from
+     */
+    domain: string
```

Comment:
Need to be optional to be passive right? 
```suggestion
    domain?: string
```

Reply:
All the commands accept `Partial<CookieOptions>`, so it's not necessary to mark it as optional here.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/24471/commits/d4e600dd589e1a6ff6b6bfcf25ca1d193a3c8fc0